specifications/api/swagger.yaml

```diff
@@ -344,27 +375,42 @@ x-stakePoolMetrics: &stakePoolMetrics
       description: Number of blocks produced by a given stake pool
 
 x-networkInformationSyncProgress: &networkInformationSyncProgress
-  <<: *percentage
+  <<: *syncProgress
+  description: |
+    Estimated synchronization progress of the node with the underlying network. Note that this may
+    change quite arbitrarily as the node switches to different shorter or longer chain forks.
 
 x-networkInformationBlockchainHeight: &networkInformationBlockchainHeight
   <<: *numberOfSlots
 
-x-networkInformationLocalHeight: &networkInformationLocalHeight
-  <<: *numberOfSlots
-
 x-networkInformationNtpStatus: &networkInformationNtpStatus
   type: object
+  description: |
+    [Network Time Protocol](https://fr.wikipedia.org/wiki/Network_Time_Protocol) information of the server.
+
+    **Important:**
+
+    - This piece of information only makes sense when the server runs on the same host machine as the node.
+    - `drift` is only returned if the underlying status is `available`.
+    - This work is still in progress; currently the status will always be set to `unavailable`.
   required:
     - status
   properties:
     status:
       type: string
       enum:
-        - available
-        - pending
         - unavailable
+        # Work in progress, always return `unavailable` at the moment.
+        #
+        # - available
+        # - pending
     drift:
       type: object
+      description: |
+        <span style="position: relative; left: 35px; top: -21px; vertical-align: middle; background-color: rgba(142, 142, 220, 0.05); color: rgba(50, 50, 159, 0.9); margin: 0 5px; padding: 0 5px; border: 1px solid rgba(50, 50, 159, 0.1); line-height: 20px; font-size: 13px; border-radius: 2px;">
+        <strong>if:</strong> status == available
+        </span><br/>
+        Drift offset of the local clock
```

Comment:
This line is missing a full stop. Should be: 
`Drift offset of the local clock.`